lib/modules/posts/view_model/new_post_view_model.dart

```diff
@@ -38,16 +39,18 @@ abstract class NewPostViewModelBase with Store {
       if (!formKey.currentState!.validate()) {
         throw const FormatException('The Form is invalid');
       }
-      final Post post = await _repository.add(
+      final post = await _repository.add(
         text: textController.text,
         creationDate: DateTime.now().toString(),
       );
       textController.clear(); // empty TextEditingController
+      // ignore: use_build_context_synchronously
```

Comment:
Let's fix these ones instead of ignoring them.
the have to do it would be to have a final local variable earlier in the method like
```dart
final navigator = Navigator.of(context); 
```
before any `await`